components/comment/__tests__/__snapshots__/demo.test.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`renders ./components/comment/demo/basic.md correctly 1`] = `
 <div
-  class="ant-comment"
+  class="ant-comment-comment"
```

Comment:
`ant-comment-comment` is a strange className.

Reply:
will have a look and update tests